Mobile/app/src/main/java/com/lesenia/mobile/AccountFragment.java

```diff
@@ -62,26 +62,11 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         initViews(Objects.requireNonNull(getView()));
         getUserInfo();
 
-        usernameSubmitBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                onPressUsernameUpdateBtn();
-            }
-        });
+        usernameSubmitBtn.setOnClickListener(view1 -> onPressUsernameUpdateBtn());
 
-        emailSubmitBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                onPressEmailUpdateBtn();
-            }
-        });
+        emailSubmitBtn.setOnClickListener(view12 -> onPressEmailUpdateBtn());
 
-        pictureBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                uploadProfilePicture();
-            }
-        });
+        pictureBtn.setOnClickListener(view13 -> uploadProfilePicture());
```

Comment:
do not use name as view13 - what is difference between view1, view12 and view 13?